std/bitmanip.d

```diff
@@ -1237,17 +1237,19 @@ public:
         return hash;
     }
 
-    deprecated("Please use the constructor instead. This function will be removed in Jan 2016.")
-    /***************************************
-     * $(RED Will be deprecated in 2.068. Please use the constructor instead.)
-     */
+    // @@@DEPRECATED_2016-01@@@
+    /++
+        $(RED Deprecated. Please use the constructor instead. This will be
+              removed in January 2016.)
+      +/
+    deprecated("Please use the constructor instead.")
```

Comment:
Well, I may change the messages at some point (if nothing else, to make them shorter), but if I do, I might as well change them all at once. I'll them as-is for the moment.